server/controllers/finance/reports/journal/index.js

```diff
@@ -42,12 +43,40 @@ function postingJournalExport(req, res, next) {
   return Journal.journalEntryList(options)
     .then(rows => report.render({ rows }))
     .then((result) => {
-      res.set(result.headers).send(result.report);
+      if (result.headers.type === 'xlsx') {
+        res.xls(result.headers.filename, filter(result.report.rows));
+      } else {
+        res.set(result.headers).send(result.report);
+      }
     })
     .catch(next)
     .done();
 }
 
+function dateFormat(date) {
```

Comment:
The [csv module already has this feature](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/server/lib/renderers/csv.js#L19), with a comment from @sfount.

```js
// @TODO Discuss if this should be moved into its own library
const DATE_FORMAT = 'DD/MM/YYYY H:mm:s';
```

Since you are using the same date format here, I propose you make a new library for dealing with formats like dates and include it in both the CSV renderer and the XLSX renderer.  Otherwise, this is a duplication of code (and not very [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)).